app-drawer/app-drawer.html

```diff
@@ -45,8 +45,10 @@
 `--app-drawer-content-container` | Mixin for the drawer content container | {}
 `--app-drawer-scrim-background`  | Background for the scrim               | rgba(0, 0, 0, 0.5)
 
-@demo app-drawer/demo/demo1.html
-@demo app-drawer/demo/demo2.html
+@group App Elements
+@element app-drawer
+@demo app-drawer/demo/index.html Simple Left Drawer
```

Comment:
`app-drawer/demo/index.html` -> `app-drawer/demo/left-drawer.html`. Then we can have an index.html that links to all the demos. That way, people who clicked "demo" from the standalone version of iron-component-page can access all the demos.